tests/Oryx.Tests.Common/DockerCli.cs

```diff
@@ -147,6 +150,75 @@ public DockerRunCommandProcessResult RunAndDoNotWaitForProcessExit(DockerRunArgu
                 $"{DockerCmd} {string.Join(" ", arguments)}");
         }
 
+        public async Task<DockerRunCommandProcessResult> RunAndWaitForContainerStartAsync(DockerRunArguments dockerRunArguments)
```

Comment:
- [ ] TODO: utilize existing RunAndDoNotWaitForProcessExit method to prevent copied code